src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color 300ms, transform 300ms linear;
```

Comment:
Use variables for the main values so that you'll be able to reuse them and give them descriptive names